etna/auto/optuna/wrapper.py

```diff
@@ -0,0 +1,98 @@
+from typing import Callable
+from typing import Optional
+from typing import Sequence
+from typing import Union
+
+import optuna
+from optuna.pruners import BasePruner
+from optuna.samplers import BaseSampler
+from optuna.storages import BaseStorage
+from optuna.study import Study
+from optuna.study import StudyDirection
+from optuna.trial import Trial
+from typing_extensions import Literal
+
+from etna.auto.runner import AbstractRunner
+from etna.auto.runner import LocalRunner
+
+OptunaDirection = Literal["minimize", "maximize"]
+
+
+class Optuna:
+    """Class for encapsulate work with Optuna."""
+
+    def __init__(
+        self,
+        direction: Union[OptunaDirection, StudyDirection],
+        study_name: Optional[str] = None,
+        sampler: Optional[BaseSampler] = None,
+        storage: Optional[BaseStorage] = None,
+        pruner: Optional[BasePruner] = None,
+        directions: Optional[Sequence[Union[OptunaDirection, StudyDirection]]] = None,
+        load_if_exists: bool = True,
+    ):
+        """Init wrapper for Optuna.
+
+        Parameters
+        ----------
+        direction:
+            optuna direction
+        study_name:
+            name of study
+        sampler:
+            optuna sampler to use
+        storage:
+            storage to use
+        pruner:
+            optuna pruner
+        directions:
+            directions to optimize in case of multi-objective optimization
+        load_if_exists:
+            load study from storage if it exists or raise exception if it doesn't
+        """
+        self.sampler = sampler
```

Comment:
Why do we have to save these parameters if it isn't a child of `BaseMixin`?

Reply:
I guess we can delete them for now.